Tests/FluentAssertions.Equivalency.Specs/AssertionRuleSpecs.cs

```diff
@@ -245,6 +245,72 @@ public void When_the_runtime_type_does_match_the_equality_comparer_type_it_shoul
         act.Should().Throw<XunitException>().WithMessage("*ConcreteClassEqualityComparer*");
     }
 
+    [Fact]
+    public void When_the_compile_time_type_is_nullable_but_the_equality_comparer_type_is_not_it_should_use_the_default_mechanics()
```

Comment:
We are not using this when...should pattern anymore... Keep it functional

Same below

Reply:
E.g. `A_non_nullable_equality_comparer_can_work_is_not_used_for_a_nullable_member`

Reply:
To be honest, I simply picked the pattern from the other tests that were already within the same file.  Same thing for `act.Should().NotThrow()`.

Are you asking me to update the other tests within the same file to fit the new pattern or just the tests I added in which case the old and the new pattern will be present?

Reply:
No, we just try to make sure we use the correct style for the new tests. And then we gradually update the others.